hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/DroolsRulesEngineImpl.java

```diff
@@ -42,6 +44,7 @@
  * @author Lucas Ponce
  */
 @Singleton
+@TransactionAttribute(value= TransactionAttributeType.NOT_SUPPORTED)
```

Comment:
As the C*Impl SLSB has not tx support and the rules engine is not invoked in a different context, I think it has sense to move it as well to NOT_SUPPORTED.
In any case, calling it from a NOT_SUPPORTED just will create a new tx id, but as drools library is not XA, it won't have any significant effect.